src/components/header/Header.tsx

```diff
@@ -0,0 +1,58 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import logo from '../../../public/logo.svg';
+import smallBannerImage from '../../../public/banners/Breakpoint=Small.png';
+import mediumBannerImage from '../../../public/banners/Breakpoint=Medium.png';
+import largeBannerImage from '../../../public/banners/Breakpoint=Large.png';
+
+export const bannerImages = [
```

Comment:
🟡 consider extracting it into a separate variable, to keep the code more modular?